vizro-core/src/vizro/static/css/aggrid.css

```diff
@@ -0,0 +1,95 @@
+.ag-theme-quartz.vizro {
```

Comment:
I've had to try out a few stuff myself and added my changes here: https://github.com/mckinsey/vizro/pull/329
You can take a look - I tried to add explanations on the relevant changes.

**Main changes include:**
- **Use AG-Grid variables where possible:** I've removed some of the properties by defining them globally in the ag-grid variables. They have plenty and I really liked their documentation and examples! This way we can avoid specifying certain properties several times.
- **Use our CSS variables for sizes**: Use our spacing, font-size, etc. variables where possible. It will be easier to change these globally then :) 
- **Add property specifications from Figma**: I've added some font specifications and hover font color ones where it was missing. Would be great if you could double-check! I am not sure if they are still visible to you? To access the dev version now, you need to have a license, so let me know if this doesn't work for you anymore.

In that draft PR I only looked at the header and rows - will comment on the other bits in another comment :)